hivelocity/helpers_resource_device.go

```diff
@@ -382,9 +382,9 @@ func getBareMetalUpdatePayloadFromGroup(d *schema.ResourceData, deviceKey string
 	if value := fieldGet("ignition_id", d, deviceKey); value != nil {
 		ignitionId = int32(value.(int))
 	}
-	if value := fieldGet("private_network", d, deviceKey); value != nil {
-		privateNetwork = value.(string)
-	}
+	//if value := fieldGet("private_network", d, deviceKey); value != nil {
+	//	privateNetwork = value.(string)
+	//}
```

Comment:
I think this can be deleted since the field isn't there anymore

Reply:
I saw some other references to this field were also commented out in other sections of the code, so I thought you guys could be considering reintroducing it in the near future. But I can remove them all if that's not the case.

Reply:
Yeah it was there because at the time I thought we were going to do some private network setup in SPS where we'd be consuming the same network APIs as a client would use but in the provisioning process.

I dunno how far we are from that but better to start clean since some of it in Core is already removed.

Reply:
Got you. I'll remove those references then.

Reply:
Done :+1: